cms/plugins/appuser/user/models/Dashboard.php

```diff
@@ -0,0 +1,38 @@
+<?php namespace Appuser\User\models;
+
+use October\Rain\Database\Model;
+
+class Dashboard extends Model
+{
+    use \October\Rain\Database\Traits\Validation;
+
+    public $table = 'appuser_dashboards';
+
+    public $rules = [];
+
+    public $jsonable = [
+        'companies'
+    ];
+
+    public $belongsTo = [
+        'user' =>
+            'LibUser\UserApi\Models\User',
+            'key' => 'user_id',
+            'otherKey' => 'id',
+    ];
+
+    public $belongsToMany = [
+        'companies' => [
+            'Appentities\Company\Models\Company',
+            'table' => 'appuser_dashboards_companies',
+        ]
+    ];
+
+    public function beforeCreate()
+    {
+        if (auth()->user()) {
```

Comment:
toto by som takto nerobil, radsej by som to vzdy priradoval niekde pri vytvarani